packages/app-desktop/gui/ShareFolderDialog/ShareFolderDialog.tsx

```diff
@@ -147,7 +147,7 @@ function ShareFolderDialog(props: Props) {
 
 	useEffect(() => {
 		void ShareService.instance().refreshShares();
-	}, []);
+	}, [props.folderId]);
```

Comment:
It should be sufficient to call `void ShareService.instance().refreshShares();` just once when opening the share dialog.

Reply:
So it seems making `props.folderId` a dependency is not needed?